src/main/content/index.html

```diff
@@ -30,7 +30,8 @@ <h2>{{t.index.welcome}}<span id="username"></span>!</h2>
             <!-- Documentation Link -->
             <div class="row">
                 <div class="col-md">
-                    <p>{{t.index.intro-2-please}} <a href="/docs">{{t.index.intro-2-doc}}</a> {{t.index.intro-2-next-steps}}</p>
+                    <p>{{t.index.intro-2-please}}<a href="{{site.site_custom.index.docs_url}}">{{t.index.doc}}</a> {{t.index.intro-2-next-steps}}</p>
+                    <p>{{t.index.intro-2-please}} <a href="{{site.site_custom.index.docs_url}}">{{t.index.intro-2-doc}}</a> {{t.index.intro-2-next-steps}}</p>
```

Comment:
looks like merge conflict resolve prob, need to delete the old line 
`<p>{{t.index.intro-2-please}}<a href="{{site.site_custom.index.docs_url}}">{{t.index.doc}}</a> {{t.index.intro-2-next-steps}}</p>`